corehq/pillows/mappings/user_mapping.py

```diff
@@ -176,6 +176,30 @@
                         },
                     }
                 },
+                'last_device': {
```

Comment:
How will this be updated if the pillow isn't firing signals?

Reply:
The `UserPillow` will still pick this up. Not firing signals just means it won't trigger an immediate update to ES and won't spawn a task to sync phone number.s